network-store-client/src/main/java/com/powsybl/network/store/client/RestNetworkStoreClient.java

```diff
@@ -848,4 +891,29 @@ public void updateGrounds(UUID networkUuid, List<Resource<GroundAttributes>> gro
             AttributeFilter attributeFilter) {
         updateAll(STR_GROUND, "/networks/{networkUuid}/grounds", groundResources, attributeFilter, networkUuid);
     }
+
+    @Override
+    public Optional<ExtensionAttributes> getExtensionAttributes(UUID networkUuid, int variantNum, ResourceType resourceType, String identifiableId, String extensionName) {
+        return getExtensionAttributes("/networks/{networkUuid}/{variantNum}/identifiables/{identifiableId}/extensions/{extensionName}", networkUuid, variantNum, identifiableId, extensionName);
+    }
+
+    @Override
+    public Map<String, ExtensionAttributes> getAllExtensionsAttributesByResourceTypeAndExtensionName(UUID networkUuid, int variantNum, ResourceType resourceType, String extensionName) {
+        return getExtensionAttributesMap("/networks/{networkUuid}/{variantNum}/types/{type}/extensions/{extensionName}", networkUuid, variantNum, resourceType, extensionName);
```

Comment:
maybe better to have separate methods for loading a map
identifiableID -> namedextension
vs a map of
extensionname -> allextensions

Reply:
I'm not sure what's best here as it will duplicate this code: https://github.com/powsybl/powsybl-network-store/blob/e5d30f4a1460fe320055302ddbd40a1a9d7ab77a/network-store-client/src/main/java/com/powsybl/network/store/client/RestNetworkStoreClient.java#L129-L135  
which is exactly the same between the two map retrieval ?